x/twap/types/utils.go

```diff
@@ -37,16 +37,19 @@ func GetAllUniqueDenomPairs(denoms []string) []DenomPair {
 // SpotPriceMulDuration returns the spot price multiplied by the time delta,
 // that is the spot price between the current and last TWAP record.
 // A single second accounts for 1_000_000_000 when converted to int64.
-func SpotPriceMulDuration(sp sdk.Dec, timeDelta time.Duration) sdk.Dec {
+func SpotPriceMulDuration(sp sdk.Dec, timeDeltaMs int64) sdk.Dec {
+	return sp.MulInt64(timeDeltaMs)
+}
+
+func SpotPriceMulDurationOld(sp sdk.Dec, timeDelta time.Duration) sdk.Dec {
```

Comment:
should we add a godoc explaining why Old is still retained / its purpose? I don't see it used anywhere but I might have missed it.

Reply:
unsure of why we would use this either

Reply:
Kept it for testing and missed it during final review, going to remove now

Reply:
removed